docs/source/tutorial/bond-extension.rst

```diff
@@ -0,0 +1,130 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Tutorials : How to use the Bond extension package
+#################################################
+
+How to create a bond instrument
+*******************************
+
+There are different types of bonds, which mainly differ in the way the coupon is defined.
+In order to create a bond instrument you first have to decide what type it is.
+The bond extension package currently supports the following bond types:
+
+Fixed rate
+==========
+
+Fixed rate bonds pay a constant coupon each coupon period. The coupon is quoted on a yearly basis (per annum, p.a.), but it could be paid more frequently.
+For example, a bond could have a 2% p.a. coupon and a 6M coupon period. That would mean
+a 1% coupon is paid twice a year.
+
+Here is an example of a bond paying a 1.1% p.a. coupon with a 12M coupon period:
+
+.. code:: daml
```

Comment:
We should have the code snipplets refer to actual code we compile during CI. Have a look at how the `getting-started` tutorial includes these. The code could either live in an actual test script, or we could have a dedicated project in a `code` folder, similar to the `getting-started` one.

Reply:
The getting-started code is self-contained (in the code folder) but currently it is not built as part of the CI. I also feel that we should refer the user to either a self-contained project or a script they can run.

Reply:
I have replaced the code in the docs by references to code from our existing code base, which are compiled during CI. From a maintainability point of view I think it is great that we can do this now that the docs live in the Daml Finance repo.